content/blog/docker-essentials-for-front-end-developers/index.md

```diff
@@ -1,96 +1,99 @@
 ---
 title: Essential Docker for Front End Developers
-date: "2020-03-01"
+date: "2020-03-10"
 author: franleplant
-description:
-  "In this post we are going to cover the steps we took to create the nosleepjavascript.com blog
-  in simple, easy, steps using the latest front end technologies: Gatsby.js (JAMstack), NodeJs, React,
-  GraphQL, Javascript, GitHub Pages and GitHub Actions"
+description: "Docker has become a widely used
+  technology and chances are you are going to have to deal with it eventually, at least superficially, in your Front End
+  career. Lets cover the basic concepts and day to day useful commands you will likely use when dealing with Docker."
 tags:
   - Docker
+  - Dockerfile
   - Javascript
   - front end developers
   - NodeJs
   - Express
 ---
 
-
-- What is docker?
+## What is docker?
 
 > Docker is an open platform for developing, shipping, and running applications.
+>
 > [source][1]
 
-
-Docker lets you encapsulate your application along with all the
+[Docker][14] lets you encapsulate your application along with all its
 system dependencies into a single executable
 that can be shipped predictably into servers or clusters; or
 as development environment for other developers using different
 Operating Systems.
 
 If you are familiar with Virtual Machines (such as [Virtual Box][2]) or [Vagrant][3]
-then you should feel confortable enough with Docker.
+then you should feel comfortable enough with Docker.
 
 With Docker you can easily create something very similar to a headless virtual machine.
 You can install system dependencies, configure networks and ports, setup databases,
-fetch your code and start your app with simple configuration files (Images).
-
-Today is very common for IaaS and PaaS providers to provide first
-class support for Docker Containers.
-Examples of these providers are Google Cloud Compute,
-Microsoft Azure and Amazon AWS among others.
-You can use Kubernetes to manage a cluster composed of docens
-of Docker Containers in these providers and scale easily
+fetch your code and start your app with a simple configuration files i.e. _Dockerfile_.
+
+Today is very common for [IaaS][5] and [PaaS][6] providers to provide first
+class support for Docker containers.
+Examples of these providers are [Google Cloud Compute Engine][7],
+[Microsoft Azure][8] and [Amazon AWS][9] among others.
+You can use [Kubernetes][15] to manage a cluster composed of docens
+of Docker containers in these providers and scale easily
```

Comment:
Maybe "Cloud computing" or similar could be in the tags as well since it's related and also discussed in the post.

Reply:
done